indico/modules/events/abstracts/controllers/management.py

```diff
@@ -134,12 +134,16 @@ def _process(self):
 
         if form.validate_on_submit():
             role_data = form.roles.role_data
-            self.event.global_conveners = set(role_data['global_conveners'])
-            self.event.global_abstract_reviewers = set(role_data['global_reviewers'])
+            for principal in role_data['global_conveners']:
+                self.event.update_principal(principal, permissions={'convene_all'})
```

Comment:
I'd go for `convene_all_abstracts`.. that's clearer (not sure why we didn't do that previously)

Reply:
Also, if you rename the permissions you need SQL in the migration step to update the permissions array (hopefully postgres provides functions to replace data in an array) in each acl entry